app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::BooksController < Api::V1::BaseController
+  def index
+    books = Book.all
+    render(json: books.to_json)
```

Comment:
Use JBuilder for this, it gives you more flexibility when you are building responses.
`to_json` return only the id of the association and nothing else from the associated records.

https://github.com/rails/jbuilder